docs/CRSF.md

```diff
@@ -2,36 +2,41 @@
 
 ([back to main page](../README.md))
 
-This page describes how to set up a mLRS system for EdgeTx/OpenTx radios, such that you get the usual CRSF telemetry sensors and can use the Yaapu telemetry app.
+This page describes how to set up a mLRS system for EdgeTX/OpenT radios, so that you get the usual CRSF telemetry sensors and can use the Yaapu telemetry app.
 
-Two things need to be done, (1) the mLRS Tx module needs to be put into "CRSF mode", and (2) the flight controller needs to be set up to send a MAVLink stream with the desired MAVLink messages.
+Two things need to be done:
+1. The mLRS Tx module needs to be put into "CRSF mode"
+2. The flight controller needs to be set up to send a MAVLink stream with the desired MAVLink messages
 
-In principle, there is no specific configuration of the mLRS receiver neccessary. It is however recommended to set the receiver into "mavlink mode", as described below.
+Optional but recommended steps:
+- Set the receiver into "mavlink mode" (described below)
+- Install the LUA script on the radio which allows you to change the Tx and Rx parameters: ([LUA Script](LUA.md))
 
-Note: Any radio which supports CRSF should work. This should include many brands besides EdgeTx/OpenTx radios.
-
-Note: An ArduPilot flight controller is assumed. For PX4 and iNav it needs to be tested and seen.
+Notes:
+- Any radio which supports the CRSF protocol should work, this includes many brands besides EdgeTX/OpenTX radios.
+- An ArduPilot flight controller is assumed. PX4 and INAV needs to be tested and validated.
 
 
 ## mLRS Tx Module Setup
 
 - Tx Ch Source = crsf
+- TX Ser Baudrate = 57600
 - Tx Ser Dest = serial or serial2 (not mbridge!)
 - Tx Ser Link Mode = mavlink
 - Tx Snd RadioStat = off (yes, off!)
 
-Note: There are situations in which it can be usefull to enable "Tx Snd RadioStat", but you should do this only if you know what you are doing. You really should not need it for this setup.
+Note: There are situations in which it can be useful to enable "Tx Snd RadioStat", but you should do this only if you know what you are doing. You really should not need it for this setup.
 
 
 ## ArduPilot Setup
 
-Configuration of a serial for MAVLink v2
+Configuration of a Serial Port for MAVLink v2:
```

Comment:
this is not a title but text, reconsider uper/lower casing :)

Reply:
Now lowercase.